policyengine_us/variables/gov/local/ca/la/general_relief/la_general_relief_base_amount.py

```diff
@@ -11,9 +11,18 @@ class la_general_relief_base_amount(Variable):
     reference = "https://drive.google.com/file/d/1Oc7UuRFxJj-eDwTeox92PtmRVGnG9RjW/view?usp=sharing"
 
     def formula(spm_unit, period, parameters):
-        married = add(spm_unit, period, ["is_married"])
+        married = add(spm_unit, period, ["is_married"]) == 1
```

Comment:
how could only one person in a spmu be married?

Reply:
This is only for the where statement 

For this syntax:
When just defining `add(spm_unit, period, ["is_married"])` the married variable returns an `int` and not a `bool`

Thus when evaluating
`married & (people_eligible_based_on_immigration_status == 2)` I am pretty sure np.where is trying to evaluate:
`married == 1 & (people_eligible_based_on_immigration_status == 2)` which returns an error due to missing brackets (we are using an `add` as `is_married` is not defined for the Spm Unit) 

By defining this condition beforehand we circumvent this error (married is a bool)

Reply:
Not sure if the explanation is 100% correct but the core is:
Where statement returns an error if removing `== 1`

Reply:
Adjusted based on the conversation